isb/Dockerfile

```diff
@@ -47,7 +47,7 @@ COPY ./isamples_webui/index.html /app/isb_web/ui/
 COPY ./isamples_webui/manifest.json /app/isb_web/ui/
 COPY ./isamples_webui/robots.txt /app/isb_web/ui/
 COPY ./isamples_webui/static /app/isb_web/ui/static
-RUN echo "const _solr_base = \"$ISB_SITEMAP_PREFIX/thing/\";\n\nconst config = {\n  \"solr_url\": _solr_base + \"select\",\n  \"solr_stream\": _solr_base + \"stream\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"$ANALYTICS_DOMAIN\",\n  \"datacite_prefix\": \"$DATACITE_PREFIX\",\n  \"datacite_publisher\": \"$DATACITE_PUBLISHER\"\n};\n\nwindow.config = config;" | tee /app/isb_web/ui/config.js
+RUN echo "const _server_base = \"$ISB_SITEMAP_PREFIX/\";\n\nconst DATACITE_PUBLISHER = [\"Sesar\", \"Geome\", \"OpenContext\", \"Smithsonian\"];\n\nconst config = {\n  \"solr_url\": _server_base + \"select\",\n  \"solr_stream\": _server_base + \"stream\",\n  \"thingpage\": _server_base + \"thingpage\",\n  \"dois_draft\": _server_base + \"mint_draft_identifiers\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"$ANALYTICS_DOMAIN\",\n  \"datacite_prefix\": \"$DATACITE_PREFIX\",\n  \"datacite_publisher\": \"$DATACITE_PUBLISHER\"\n};\n\nwindow.config = config;" | tee /app/isb_web/ui/config.js
```

Comment:
@dannymandel `solr_url` and `solr_stream` should be `thing/select` and `thing/stream` because we use base URL right now.